test/controllers/splash_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class SplashControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
- [ ] Kindly remove commented piece of code and add the test code. This makes your codebase clean and professional